### ISSUE TITLE
Feature Request: Eclair plugins

### ISSUE THREAD

**Issue by orbitalturtle** [jamaljsr/polar#853]
This is more of a question than a feature request, but is it currently possible to load an Eclair plugin into an Eclair node running in Polar? Using an Eclair plugin usually looks like this: https://github.com/ACINQ/eclair?tab=readme-ov-file#plugins

**Comment by jamaljsr** [comment]
Hey @orbitalturtle I haven't done this myself but it should be possible. Off the top of my head, I can think of two approaches that may work.

1. Create a [custom docker image](https://github.com/jamaljsr/polar/blob/master/docs/custom-nodes.md) which includes your plugin's `.jar` file in the image and add the new image in Polar. Create a new network including one or more nodes which use your custom image. Before starting the network, edit the command line args by right-clicking on the node then clicking on Advanced Options. In here you should be able to include the path to your `.jar` file to the startup command. Just make sure you use the path within the docker image's file system.
2. I'm not sure if this will work, but it may be worth a try as it doesn't require creating a custom Docker image. First create a new network with just and bitcoind and an Eclair node. Start the network one then stop it after the nodes successfully start up. On your machine's file system, open up the dir `~/.polar/networks/1/volumes/eclair/alice/` and you should see a bunch of files that Eclair creates on startup. This dir maps to the path `/home/eclair/.eclair` inside of the Docker container that the Eclair node runs in. Copy your `.jar` file into this folder. Back in Polar, right-click on the Eclair node and click Advanced Options. Pre-fill the default command and add the arg `/home/eclair/.eclair/<plugin1.jar>` using the actual name of your plugin `.jar` file. Save the options then try to start the network. Check the logs for the eclair node to see if it loads the plugin successfully or fails.

I hope this helps point you in the right direction. If you have any further questions, please feel free to reply here.

**Comment by orbitalturtle** [comment]
Thank you @jamaljsr! The first option worked for me with a couple of changes. Figure I'll outline what I did if this is helpful for anyone else: 
- In addition to copying the plugin jar via the Dockerfile, I needed to make a few updates to the custom Docker image for the latest version of Eclair. Eclair-node-gui is now eclair-front, and maybe one or two other small things IIRC. (I'd be happy to add these updates in a PR if that sounds helpful.)
- Initially I tried loading the BOLT 12 plugin jar to /home/eclair in the docker filesystem. Weirdly, loading the BOLT 12 plugin as I normally would, `i.e. polar-eclair /home/eclair/bolt12-tip-jar-0.9.1-SNAPSHOT.jar  ...<CONFIG_OPTIONS>`, it wasn't detected by eclair. Once I added the necessary config options, it still didn't seem to detect that I was passing in a plugin jar, BUT Eclair errored saying it expected the plugin to be located at `/app/offer` . So if I changed `bolt12-tip-jar-0.9.1-SNAPSHOT.jar` to `offer` and moved it to the `/app` folder, it worked just fine.
